cfme/cloud/provider.py

```diff
@@ -53,6 +53,7 @@
             }
         )),
         ("api_version", AngularSelect("api_version"), {"appeared_in": "5.5"}),
+        ('sec_protocol', {version.LOWEST: None, '5.5': AngularSelect("security_protocol")}),
```

Comment:
Probably better to do like the line above here, I can't recall what setting it to ```None``` will do. It _should_ be OK, but may have undesired consequences.